OrbitGrpcProtos/capture.proto

```diff
@@ -78,6 +78,22 @@ message CallstackSample {
   uint64 timestamp_ns = 5;
 }
 
+message InternedTracepointInfo {
+  uint64 key = 1;
+  TracepointInfo intern = 2;
+}
+
+message TracepointSample {
+  int32 pid = 1;
+  int32 tid = 2;
+  int64 time = 3;
+  int32 cpu = 4;
+  oneof tracepoint_info_or_key {
+    TracepointInfo tracepoint_info = 5;
+    uint64 tracepoint_key = 6;
```

Comment:
`tracepoint_info_key`

Reply:
Thanks for pointing that out! I will change it immediately.